stan/math/prim/fun/inv_Phi.hpp

```diff
@@ -21,13 +22,15 @@ namespace math {
  *
  * \f$\Phi(x) = \int_{-\infty}^x \mbox{\sf Norm}(x|0, 1) \ dx = p\f$
  *
- * Algorithm first derived in 2003 by Peter Jon Aklam at
- * http://home.online.no/~pjacklam/notes/invnorm/
+ * Algorithm described in Wichura, M. J. (1988)
```

Comment:
I'd like to see the algorithm reference up in the public-facing (non-internal) doc.  The internal function can have minimal doc.  The definition of Phi isn't necessary, but if it's included, it should be pointed out that the function is computing its inverse.